### PR TITLE
(MAINT) Update changelog for 1.4.1 release

### PR DESCRIPTION
**Pull request by jpinsonault**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 1.4.1
+
+This is bugfix release. It fixes a single issue
```
Comment (rlinehan) [inline CHANGELOG.md]: typo, missing an `a`

Comment (jpinsonault) [inline CHANGELOG.md]: ha, woops

**Comment by jpinsonault** [comment]
updated

**Comment by rlinehan** [comment]
🚢

**Pull request merged**